app/controllers/api/v2/hosts_controller.rb

```diff
@@ -172,6 +174,25 @@ def destroy
         process_response @host.destroy
       end
 
+      def_param_group :bulk_host_ids do
+        param :organization_id, :number, :required => true, :desc => N_("ID of the organization")
+        param :included, Hash, :desc => N_("Hosts to include in the action"), :required => true, :action_aware => true do
+          param :search, String, :required => false, :desc => N_("Search string describing which hosts to perform the action on")
+          param :ids, Array, :required => false, :desc => N_("List of host ids to perform the action on")
+        end
+        param :excluded, Hash, :desc => N_("Hosts to explicitly exclude in the action."\
+                                           " All other hosts will be included in the action,"\
+                                           " unless an included parameter is passed as well."), :required => true, :action_aware => true do
+          param :ids, Array, :required => false, :desc => N_("List of host ids to exclude and not perform the action on")
+        end
+      end
+
+      api :DELETE, "/hosts/bulk/", N_("Delete multiple hosts")
+      param_group :bulk_host_ids
+      def bulk_destroy
```

Comment:
I am ok with this but wonder if we should have a separate BulkHostController? I am thinking we may have a lot more bulk hosts actions in the future and overloading this controller imo isnot ideal.

Reply:
I'll have to think.. I don't know how many of the actions will actually need to use hosts controller. Delete is special because it affects the host object itself.  hmm

Reply:
Updated to move to new controller. :+1: